heat/core/dndarray.py

```diff
@@ -1586,20 +1678,6 @@ def is_distributed(self):
         """
         return self.split is not None and self.comm.is_distributed()
 
-    def item(self):
```

Comment:
why is this deleted? is this somewhere else that im not seeing?

Reply:
Because it was a duplicate implementation, item() is already defined in line 173.